saleor/dashboard/order/forms.py

```diff
@@ -146,12 +149,14 @@ def cancel_line(self):
 
 class ChangeQuantityForm(forms.ModelForm):
     quantity = QuantityField(
-        label=pgettext_lazy('Change quantity form label', 'Quantity'),
         validators=[MinValueValidator(1)])
 
     class Meta:
         model = OrderLine
         fields = ['quantity']
+        labels = {
+            'quantity': pgettext_lazy(
+                'Quantity', 'Quantity')}
```

Comment:
I think it would be better to have "Integer number" or "Decimal number" as context (whichever is true).